packages/block-editor/src/components/list-view/style.scss

```diff
@@ -339,6 +339,28 @@
 	.block-editor-list-view-block-select-button__sticky {
 		line-height: 0;
 	}
+
+	.block-editor-list-view-block-select-button__images {
+		display: flex;
+		align-items: center;
+		flex-direction: row;
```

Comment:
`row` is the default direction so we shouldn't need to specify it here, unless I'm missing something? `align-items` also shouldn't be needed if the images are always the same size (and if they aren't, would it be better to leave it as `stretch` so they all take up the same height?)

Reply:
Oh, thank you! Those were stray rules left over from hacking around, I'll remove them 🙂